packages/core/src/checkout-buttons/strategies/googlepay/googlepay-button-options.ts

```diff
@@ -1,5 +1,8 @@
-import { BuyNowCartRequestBody } from '../../../cart';
-import { ButtonColor, ButtonType } from '../../../payment/strategies/googlepay';
+import {
+    ButtonColor,
+    ButtonType,
+    GooglePayBuyNowInitializeOptions,
+} from '../../../payment/strategies/googlepay';
```

Comment:
Can I ask why are we moving this interface into the payment folder?

Reply:
I didn't move. I just added GooglePayBuyNowInitializeOptions interface specifically for googlepay. For Example in braintreevenmo we have the same.